Ccarfi alphatweet blog post (#11829)
unlock-protocol/unlock: unlock-protocol-com/blog/alphatweet-announce.md

---
title: Unlock Labs Launches the ALPHAtweet Experiment
authorName: Christopher Carfi
publishDate: May 9, 2023
description: ALPHAtweet is an experiment in web3 business models where creators and their community both share in the upside of creating and sharing content across social networks.
image: /images/blog/alphatweet/alphatweet-share.png
---

![alphatweet-share.png](/images/blog/alphatweet/alphatweet-share.png)

_**ALPHA by Unlock Labs** is an initiative to explore new business models and opportunities enabled by blockchain technologies and using Unlock Protocol. It is an iterative series of experiments with the intended goals of learning and sharing information and experiences as part of the open-source Unlock Protocol project._

## This experiment is called ALPHAtweet

> ALPHAtweet enables you and your followers make money through your content and sharing. It’s powered by Unlock Protocol. Going a bit deeper into this experiment, an “Alpha” is a piece of paid content that is being shared, where **both the creator and the sharer participate in the benefits of sharing when someone unlocks the Alpha**.

Reviewer: I am not sure but I think this sentence misses a "to"?

"ALPHAtweet enables you and your followers **to** make money through your content and sharing."

> ALPHAtweet enables you and your followers make money through your content and sharing. It’s powered by Unlock Protocol. Going a bit deeper into this experiment, an “Alpha” is a piece of paid content that is being shared, where **both the creator and the sharer participate in the benefits of sharing when someone unlocks the Alpha**.

Author: ```suggestion
ALPHAtweet enables you and your followers to make money through your content and sharing. It’s powered by Unlock Protocol. Going a bit deeper into this experiment, an “Alpha” is a piece of paid content that is being shared, where **both the creator and the sharer participate in the benefits of sharing when someone unlocks the Alpha**.
```

While this first experiment emphasizes sharing on Twitter, the magic is that these items of Alpha can be shared *anywhere* by way of share links and, in every case, the creators and sharers **both** have the opportunity to be compensated across *any* platform, every time an Alpha is shared. 🤯 

> This is not something that could be done before today.

Reviewer: ```suggestion
This is not something that could not be done before today.
```

Author: @julien51 with that addition, the sentence has a double negative (not...not). are you suggesting i move the "not" to the new place after "could..." and remove the earlier one? or...?

Reviewer: You are correct!

Creators set their own price for unlocking their exclusive content, the Alpha. They also set a referral percentage bonus that is split with those who share or amplify the content.

In this model, the creator can set the referral percentage bonus to any level they want, based on the creator’s own goals. For example:

> - A creator could set the referral bonus to 100% (sharing everything) to optimize for reach, awareness, exposure, and virality of their content, where those doing the sharing get 100% of the fees collected when the creator’s content is unlocked.

Author: ```suggestion
- A creator could set the referral bonus to 100% (giving **all** revenue received to the sharer) to optimize for reach, awareness, exposure, and virality of their content, which means that those doing the sharing get 100% of the fees collected when the creator’s content is unlocked.
```

- Or, the creator could set the referral bonus somewhere in the middle (50/50 or any other amount; it’s up to the creator). We may find out that ALPHAtweet and this model work best when set to something like 20/80 or 10/90 where the distribution is heavily incentivized (but we don’t know — that’s why it’s an experiment!).

Here’s a walkthrough diagram of how money flows to pay both creators and sharers.

<div style="position: relative; overflow: hidden; width: 100%; padding-top: 0%;"><iframe src="https://docs.google.com/presentation/d/e/2PACX-1vQ-m3vul5u2NS39QptD70hBy1ib3AzNGWaSO0Znk7sigAuYZayhN9MdR5MWJU_hMT5cqyJKUsiyh8dc/embed?start=false&loop=false&delayms=60000" frameborder="0" width="680" height="411" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe></div>

The revenue sharing mechanism is the big deal here. Creators set their terms, and sharers can participate in the upside. (It’s also completely configurable, so the platforms could participate in the upside, too.)

> At the outset, we’re starting with a small cohort of creators including (names) as we shake out bugs and get feedback on the overall experience. That said, *anyone* can choose to unlock, share, and amplify Alpha (and participate in rewards!), and *everyone* can view the snippets/teasers of Alpha.

Author: ```suggestion
At the outset, we’re starting with a small cohort of creators as we shake out bugs and get feedback on the overall experience. That said, *anyone* can choose to unlock, share, and amplify Alpha (and participate in rewards!), and *everyone* can view the snippets/teasers of Alpha.
```

Want to see what ALPHAtweet looks like in action? Here’s what the experience looks like for a sharer of Alpha. (Thanks to Michael Joseph for the video walkthrough!)

<div style="position: relative; overflow: hidden; width: 100%; padding-top: 56.25%;"><iframe style="position: absolute; top: 0; left: 0; bottom: 0; right: 0; width: 100%; height: 100%;" src="https://www.youtube.com/embed/sUTaz7-MakY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>

Want to check it out today? Check out these posts from some Alpha creators:

- Michael C. Joseph, "[Coordinape Deep Dive](https://twitter.com/michaelcjoseph/status/1655633679794962432)"
- Julien Genestoux, "[Layers of (De)centralization](https://alphatweet.xyz/posts/ed023c92-8dfd-4060-984c-bc5acdd02436)"
- Stephen Wolfson, "[The Complex World of Style, Copyright, and Generative AI](https://twitter.com/ccarfi/status/1655662240006295553)" (via Creative Commons)

> - five

Author: ```suggestion
```

---

> title: Unlock Labs launches the ALPHAtweet experiment

Author: ```suggestion
title: Unlock Labs Launches the ALPHAtweet Experiment
```

authorName: Christopher Carfi
publishDate: May 9, 2023
description: ALPHAtweet is an experiment in web3 business models where creators and their community both share in the upside of creating and sharing content across social networks.
image: /images/blog/alphatweet/alphatweet-share.png
---

> ![alphatweet-share.png](ALPHAtweet%20blog%20post%209bce8f1d01824456aecf9c4a9c4e4c88/alphatweet-share.png)

Author: ```suggestion
![alphatweet-share.png](/images/blog/alphatweet/alphatweet-share.png)
```

> **ALPHA by Unlock Labs** is an initiative to explore new business models and opportunities enabled by blockchain technologies and using Unlock Protocol. It is an iterative series of experiments with the intended goals of learning and sharing information and experiences as part of the open-source Unlock Protocol project.*

Author: ```suggestion
_**ALPHA by Unlock Labs** is an initiative to explore new business models and opportunities enabled by blockchain technologies and using Unlock Protocol. It is an iterative series of experiments with the intended goals of learning and sharing information and experiences as part of the open-source Unlock Protocol project._
```

> ***ALPHA by Unlock Labs** is an initiative to explore new business models and opportunities enabled by blockchain technologies and using Unlock Protocol. It is an iterative series of experiments with the intended goals of learning and sharing information and experiences as part of the open-source Unlock Protocol project.*

Author: ```suggestion
**ALPHA by Unlock Labs** is an initiative to explore new business models and opportunities enabled by blockchain technologies and using Unlock Protocol. It is an iterative series of experiments with the intended goals of learning and sharing information and experiences as part of the open-source Unlock Protocol project.*
```

Creators set their own price for unlocking their exclusive content, the Alpha. They also set a referral percentage bonus that is split with those who share or amplify the content.

In this model, the creator can set the referral percentage bonus to any level they want, based on the creator’s own goals. For example:

- A creator could set the referral bonus to 100% (sharing everything) to optimize for reach, awareness, exposure, and virality of their content, where those doing the sharing get 100% of the fees collected when the creator’s content is unlocked.
- Or, the creator could set the sharing referral bonus percentage to 0% and take all fees, but then there would be no economic benefit to those who do the sharing (which, to be fair, is the way sharing on social works today).
- Or, the creator could set the referral bonus somewhere in the middle (50/50 or any other amount; it’s up to the creator). We may find out that ALPHAtweet and this model work best when set to something like 20/80 or 10/90 where the distribution is heavily incentivized (but we don’t know — that’s why it’s an experiment!).

Here’s a walkthrough diagram of how money flows to pay both creators and sharers.

> <div style="position: relative; overflow: hidden; width: 100%; padding-top: 0%;"><iframe src="https://docs.google.com/presentation/d/e/1uRzAtKa4_Hm7yXbQavIwImCCSJQCOmANi1G3hp1FSKU/embed?start=false&loop=false&delayms=3000" frameborder="0" width="680" height="411" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe></div>

Author: ```suggestion
<div style="position: relative; overflow: hidden; width: 100%; padding-top: 0%;"><iframe src="https://docs.google.com/presentation/d/e/2PACX-1vQ-m3vul5u2NS39QptD70hBy1ib3AzNGWaSO0Znk7sigAuYZayhN9MdR5MWJU_hMT5cqyJKUsiyh8dc/embed?start=false&loop=false&delayms=60000" frameborder="0" width="680" height="411" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe></div>
```

The revenue sharing mechanism is the big deal here. Creators set their terms, and sharers can participate in the upside. (It’s also completely configurable, so the platforms could participate in the upside, too.)

At the outset, we’re starting with a small cohort of creators including (names) as we shake out bugs and get feedback on the overall experience. That said, *anyone* can choose to unlock, share, and amplify Alpha (and participate in rewards!), and *everyone* can view the snippets/teasers of Alpha.

Want to see what ALPHAtweet looks like in action? Here’s what the experience looks like for a sharer of Alpha. (Thanks to Michael Joseph for the video walkthrough!)

> [AlphaTweet - Sharer Journey.mp4](ALPHAtweet%20blog%20post%209bce8f1d01824456aecf9c4a9c4e4c88/AlphaTweet_-_Sharer_Journey.mp4)

Author: ```suggestion
<div style="position: relative; overflow: hidden; width: 100%; padding-top: 56.25%;"><iframe style="position: absolute; top: 0; left: 0; bottom: 0; right: 0; width: 100%; height: 100%;" src="https://www.youtube.com/embed/sUTaz7-MakY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
```

Want to check it out today? Check out these posts from some Alpha creators:

> - one

Author: ```suggestion
- Michael C. Joseph, "[Coordinape Deep Dive](https://twitter.com/michaelcjoseph/status/1655633679794962432)"
```

> - two

Author: ```suggestion
- Julien Genestoux, "[Layers of (De)centralization](https://alphatweet.xyz/posts/ed023c92-8dfd-4060-984c-bc5acdd02436)"
```

> - three

Author: ```suggestion
- ...Stephen Wolfson, "The Complex World of Style, Copyright, and Generative AI(https://twitter.com/ccarfi/status/1655662240006295553)" (via Creative Commons)
```

> - ...Stephen Wolfson, "[The Complex World of Style, Copyright, and Generative AI](https://twitter.com/ccarfi/status/1655662240006295553)" (via Creative Commons)

Author: ```suggestion
- Stephen Wolfson, "[The Complex World of Style, Copyright, and Generative AI](https://twitter.com/ccarfi/status/1655662240006295553)" (via Creative Commons)
```

> - four

Author: ```suggestion
```

> - four

Author: ```suggestion
```

> - five

Author: ```suggestion
```
